CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## 4.0.6
+  - Support for non-AWS endpoints
```

Comment:
This PR adds new functionality, so instead of a patch/bugfix version bump (4.0.5 -> 4.0.6) this needs to be a minor bump (4.0.5 -> 4.1.0)